integration-tests/src/test/java/org/zowe/apiml/gatewayservice/VersionEndpointTest.java

```diff
@@ -13,14 +13,15 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.zowe.apiml.util.config.ConfigReader;
+import org.zowe.apiml.util.categories.TestsNotMeantForZowe;import org.zowe.apiml.util.config.ConfigReader;
 import org.zowe.apiml.util.config.GatewayServiceConfiguration;
 
 import static io.restassured.RestAssured.given;
 import static org.apache.http.HttpStatus.SC_OK;
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.Matchers.is;
 
+@TestsNotMeantForZowe
```

Comment:
why is this test not running on Zowe?

Reply:
I believe that it is because the deployment on CA32 doesn't contain the necessary files

Reply:
Version endpoints shows versions "Unknown" even files aren't exist.

Reply:
Yes, but then the success path in the tests fails.

Reply:
Maybe the version test for success would make sense in the Nightly builds? There we have the full Zowe instance.